### PR TITLE
webex: fix missing libraries

### LINKED ISSUE #227055: webex: application doesn't start
### Describe the bug
The application window doesn't appear, if I run strace webex, I get https://0x0.st/H8YO.out
last few lines:
`
write(3, "2023-04-19T12:20:21.519Z Found 1"..., 77) = 77
write(3, "2023-04-19T12:20:21.519Z \n", 26) = 26
write(3, "2023-04-19T12:20:21.519Z Directo"..., 171) = 171
write(3, "2023-04-19T12:20:21.519Z \n", 26) = 26
write(3, "2023-04-19T12:20:21.519Z Examini"..., 122) = 122
write(3, "2023-04-19T12:20:21.519Z Locatio"..., 92) = 92
write(3, "2023-04-19T12:20:21.519Z Conside"..., 187) = 187
write(3, "2023-04-19T12:20:21.519Z \n", 26) = 26
write(3, "2023-04-19T12:20:21.519Z *******"..., 57) = 57
write(3, "2023-04-19T12:20:21.519Z \n", 26) = 26
write(3, "2023-04-19T12:20:21.519Z Explici"..., 139) = 139
write(3, "2023-04-19T12:20:21.519Z \n", 26) = 26
write(3, "2023-04-19T12:20:21.519Z Marked "..., 66) = 66
readlink("/nix", 0x7ffe15abbaa0, 1023)  = -1 EINVAL (Invalid argument)
readlink("/nix/store", 0x7ffe15abbaa0, 1023) = -1 EINVAL (Invalid argument)
readlink("/nix/store/iycd2x0yqavic4nm973yhpknm7jvaiaf-webex-43.2.0.25211", 0x7ffe15abbaa0, 1023) = -1 EINVAL (Invalid argument)
readlink("/nix/store/iycd2x0yqavic4nm973yhpknm7jvaiaf-webex-43.2.0.25211/opt", 0x7ffe15abbaa0, 1023) = -1 EINVAL (Invalid argument)
readlink("/nix/store/iycd2x0yqavic4nm973yhpknm7jvaiaf-webex-43.2.0.25211/opt/Webex", 0x7ffe15abbaa0, 1023) = -1 EINVAL (Invalid argument)
readlink("/nix/store/iycd2x0yqavic4nm973yhpknm7jvaiaf-webex-43.2.0.25211/opt/Webex/lib", 0x7ffe15abbaa0, 1023) = -1 EINVAL (Invalid argument)
faccessat2(AT_FDCWD, "/nix/store/iycd2x0yqavic4nm973yhpknm7jvaiaf-webex-43.2.0.25211/opt/Webex/lib/", F_OK, AT_EACCESS) = 0
clone(child_stack=NULL, flags=CLONE_CHILD_CLEARTID|CLONE_CHILD_SETTID|SIGCHLD, child_tidptr=0x7f72d1f51a10) = 116819
write(3, "2023-04-19T12:20:21.520Z Process"..., 85) = 85
write(3, "2023-04-19T12:20:21.520Z \n", 26) = 26
write(3, "2023-04-19T12:20:21.520Z *** CLE"..., 52) = 52
write(3, "2023-04-19T12:20:21.520Z \n", 26) = 26
write(3, "2023-04-19T12:20:21.520Z Exiting"..., 43) = 43
close(3)                                = 0
openat(AT_FDCWD, "/home/tei/.local/share/Webex/", O_RDONLY|O_NONBLOCK|O_CLOEXEC|O_DIRECTORY) = 3
newfstatat(3, "", {st_mode=S_IFDIR|0700, st_size=4096, ...}, AT_EMPTY_PATH) = 0
close(3)                                = 0
openat(AT_FDCWD, "/home/tei/.local/share/Webex/hostLogs/", O_RDONLY|O_NONBLOCK|O_CLOEXEC|O_DIRECTORY) = 3
newfstatat(3, "", {st_mode=S_IFDIR|0700, st_size=4096, ...}, AT_EMPTY_PATH) = 0
close(3)                                = 0
openat(AT_FDCWD, "/home/tei/.local/share/Webex/hostLogs/webexhost.txt", O_WRONLY|O_CREAT|O_APPEND, 0666) = 3
lseek(3, 0, SEEK_END)                   = 292
newfstatat(AT_FDCWD, "/etc/localtime", {st_mode=S_IFREG|0444, st_size=3664, ...}, 0) = 0
newfstatat(3, "", {st_mode=S_IFREG|0644, st_size=292, ...}, AT_EMPTY_PATH) = 0
newfstatat(AT_FDCWD, "/etc/localtime", {st_mode=S_IFREG|0444, st_size=3664, ...}, 0) = 0
newfstatat(AT_FDCWD, "/etc/localtime", {st_mode=S_IFREG|0444, st_size=3664, ...}, 0) = 0
write(3, "2023-04-19 13:20:21 ... AFTER EN"..., 154) = 154
close(3)                                = 0
exit_group(0)                           = ?
+++ exited with 0 +++
`

### Steps To Reproduce
1. install webex (from nixos-unstable)
2. run webex

### Expected behavior
webex starts

### Notify maintainers
@uvnikita

### Metadata
```
- system: `"x86_64-linux"`
 - host os: `Linux 6.1.24, NixOS, 23.05 (Stoat), 23.05.20230417.f00994e`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.15.0`
 - channels(root): `"nixos, unstable"`
 - channels(tei): `"nixos, unstable"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/unstable`
```

### PR DESCRIPTION
**Pull request by uvNikita**
Fixes missing libcrypt.so.1 dependency.

Might also fix #227055

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/webex/default.nix [hunk 1]
```diff
@@ -93,6 +94,7 @@ stdenv.mkDerivation rec {
     libxcb
     libxkbcommon
     libxcrypt
+    libxcrypt-legacy
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/instant-messengers/webex/default.nix:97]: You probably only want one libxcrypt

**Comment by uvNikita** [comment]
@SuperSandro2000 good point, changed :+1:

**Comment by SuperSandro2000** [comment]
Please add the following diff which shows me the license agreement pop up:

```diff
diff --git a/pkgs/applications/networking/instant-messengers/webex/default.nix b/pkgs/applications/networking/instant-messengers/webex/default.nix
index 6320c1309d6..4487ba18985 100644
--- a/pkgs/applications/networking/instant-messengers/webex/default.nix
+++ b/pkgs/applications/networking/instant-messengers/webex/default.nix
@@ -49,6 +49,7 @@
 , xcbutilrenderutil
 , xcbutilwm
 , p7zip
+, tbb
 , wayland
 , libXScrnSaver
 }:
@@ -62,6 +63,10 @@
     sha256 = "c58a0da26c8f64302cc612c60980dbd68c074d6d8a567b3d870d7d6d06b420ad";
   };

+  nativeBuildInputs = [
+    p7zip
+  ];
+
   buildInputs = [
     alsa-lib
     at-spi2-atk
@@ -110,7 +115,7 @@
     xcbutilkeysyms
     xcbutilrenderutil
     xcbutilwm
-    p7zip
+    tbb
     wayland
   ];
```

**Comment by uvNikita** [comment]
@SuperSandro2000 done, good catch!

**Pull request merged**